source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,214 @@
+###########################
+Enterprise Deployment Guide
+###########################
+
+================
+About This Guide
+================
+
+********
+Audience
+********
+
+IT Leaders, System Administrators, and/or Project Managers
+
+.. Note::
+    Some features described in this guide are only available to Enterprise Edition E10 and/or E20. We will highlight specific licenses needed whenever applicable.
+
+*******************
+Learning Objectives
+*******************
+
+This guide provides foundational information necessary when developing a plan for a proof of concept and/or production-level enterprise deployment of the Mattermost on-premises edition. 
+
+It will not dive into the technical implementation, but will reference any relevant documentation.
+
+By the end of this guide you should have an understanding of the high-level requirements for a successful Mattermost Enterprise deployment. 
+
+This includes:
+
+- Choosing how to deploy the Mattermost Server application
+- Migrating from other ChatOps platforms
+- Integrating Mattermost with existing Single sign-on (SSO) providers
+- First steps/best practices for onboarding users
+
+*******
+Support
+*******
+
+If at any point you need additional help, we are ready to assist, just reach out using any of these methods:
+
+- `Community <https://community.mattermost.com/>`_ - Our entire organization uses Mattermost, plus you’ll have the support of one of the best open source communities around
+- Documentation - We link to many parts of our `administrator  documentation <https://docs.mattermost.com/guides/administrator.html>`_ in this guide but encourage you to take a look at `all our documentation <https://docs.mattermost.com>`_
+- Enterprise Support  - If you are an Enterprise Edition subscriber, you may open a support ticket in the Enterprise Edition Support portal
+
+.. Recommendation::
+    Visit the `Mattermost Community server <https://community.mattermost.com/>`_ and create an account. Join channel `Ask Anything <https://community.mattermost.com/core/channels/ask-anything>`_ to not only experience Mattermost right away but also for support if you get stuck at any point in this guide.
+
+================================
+Before You Begin Your Deployment
+================================
+
+********************************
+What Makes Mattermost Different?
+********************************
+
+Mattermost is a high-trust messaging platform for enterprise use. As enterprises come in all shapes and sizes, Mattermost has been built to provide an optimal experience for all customers. 
+
+Taking this a step further, the emergence of ChatOps means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of `integrations <https://integrations.mattermost.com>`_, extensions, and customization.
```

Comment:
Comment: Should we elaborate on ChatOps - it's very important imo and maybe linking to some content around Chat/DevOps would be helpful to drive interest?

Reply:
Great idea! I'll make it link to https://mattermost.com/chatops/